sdk/boms/azure-spring-boot-bom/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 3.10.1 (2021-11-18)
+This release is compatible with Spring Boot 2.5.0 - 2.5.4.
```

Comment:
Fix azure-sdk-bom's version error: It should be 1.0.5 instead of 1.0.4,

Reply:
updated